SimPPS/RPDigiProducer/interface/DeadChannelsManager.h

```diff
@@ -0,0 +1,44 @@
+#ifndef SimPPS_RPDigiProducer_DEAD_CHANNELS_MANAGER
+#define SimPPS_RPDigiProducer_DEAD_CHANNELS_MANAGER
```

Comment:
As the implementation is in `plugins`, the header should go to `plugins` as well (same for the other headers that have the implementation in `plugins`).